tests/testsuite/path.rs

```diff
@@ -330,33 +330,27 @@ fn deep_dependencies_trigger_rebuild() {
     //
     // We base recompilation off mtime, so sleep for at least a second to ensure
     // that this write will change the mtime.
-    File::create(&p.root().join("baz/src/baz.rs"))
-        .unwrap()
-        .write_all(br#"pub fn baz() { println!("hello!"); }"#)
-        .unwrap();
-    sleep_ms(1000);
+    p.write_file("baz/src/baz.rs", "pub fn baz() {}");
     p.cargo("build")
         .with_stderr(&format!(
-            "[COMPILING] baz v0.5.0 (CWD/baz)\n\
```

Comment:
How come these outputs changed?

Reply:
Oops! I think I messed something up, wasn't intended to change